src/datafactory/azext_datafactory/generated/_help.py

```diff
@@ -0,0 +1,1166 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['datafactory factory'] = """
+    type: group
+    short-summary: datafactory factory
+"""
+
+helps['datafactory factory list'] = """
+    type: command
+    short-summary: Lists factories under the specified subscription.
+    examples:
+      - name: Factories_ListByResourceGroup
+        text: |-
+               az datafactory factory list --resource-group "exampleResourceGroup"
+"""
+
+helps['datafactory factory show'] = """
+    type: command
+    short-summary: Gets a factory.
+    examples:
+      - name: Factories_Get
+        text: |-
+               az datafactory factory show --name "exampleFactoryName" --resource-group "exampleResourceGroup"

```

Comment:
Recommend to use `MyFactoryName` and `MyResourceGroup` to follow the example value pattern.

Reply:
This has not been fixed yet, the outdated status is outdated just because I changed the eof mark.

Reply:
This should've been fixed, please check again. thx